apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -288,6 +288,240 @@ static void itc_wifi_manager_connect_disconnect_ap_p(void)
+ * @precondition     none
+ * @postcondition    none
+ */
+static void itc_wifi_manager_get_config_n(void)
+{
+	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
+	ret = wifi_manager_init(&wifi_callbacks);
+	TC_ASSERT_EQ("wifi_manager_init", ret, WIFI_MANAGER_SUCCESS);
+
+	wifi_manager_ap_config_s config;
+	wifi_manager_ap_config_s gconfig;
+
+	config.ssid_length = strlen(TEST_SSID);
+	config.passphrase_length = strlen(TEST_PASSWORD);
+	strncpy(config.ssid, TEST_SSID, config.ssid_length + 1);
+	strncpy(config.passphrase, TEST_PASSWORD, config.passphrase_length + 1);
+	config.ap_auth_type = (wifi_manager_ap_auth_type_e)TEST_AUTH_TYPE;
+	config.ap_crypto_type = (wifi_manager_ap_crypto_type_e)TEST_CRYPTO_TYPE;
+	printf("AP config: %s(%d), %s(%d), %d %d\n", config.ssid, config.ssid_length, config.passphrase, \
+			config.passphrase_length, config.ap_auth_type, config.ap_crypto_type);
+
+	ret =  wifi_manager_get_config(&gconfig);
+	TC_ASSERT_NEQ_CLEANUP("wifi_manager_get_config", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+
+	ret = wifi_manager_deinit();
+	TC_ASSERT_EQ("wifi_manager_deinit", ret, WIFI_MANAGER_SUCCESS);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @testcase         itc_wifi_manager_get_remove_config_p
+ * @brief            check initialize, connect,, save ap config, get ap config, remove ap config, de-initalization operations of wifi manager
+ * @scenario         check initialize, connect, , save ap config, get ap config, remove ap config, de-initalization operations of wifi manager
+ * @apicovered       wifi_manager_init, wifi_manager_save_config, wifi_manager_get_config, wifi_manager_remove_config wifi_manager_deinit
+ * @precondition     none
+ * @postcondition    none
+ */
+static void itc_wifi_manager_get_remove_config_p(void)
+{
+	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
+	ret = wifi_manager_init(&wifi_callbacks);
+	TC_ASSERT_EQ("wifi_manager_init", ret, WIFI_MANAGER_SUCCESS);
+
+	wifi_manager_ap_config_s config;
+	wifi_manager_ap_config_s gconfig;
+
+	config.ssid_length = strlen(TEST_SSID);
+	config.passphrase_length = strlen(TEST_PASSWORD);
+	strncpy(config.ssid, TEST_SSID, config.ssid_length + 1);
+	strncpy(config.passphrase, TEST_PASSWORD, config.passphrase_length + 1);
+	config.ap_auth_type = (wifi_manager_ap_auth_type_e)TEST_AUTH_TYPE;
+	config.ap_crypto_type = (wifi_manager_ap_crypto_type_e)TEST_CRYPTO_TYPE;
+	printf("AP config: %s(%d), %s(%d), %d %d\n", config.ssid, config.ssid_length, config.passphrase, \
+			config.passphrase_length, config.ap_auth_type, config.ap_crypto_type);
+
+	ret =  wifi_manager_save_config(&config);
+	TC_ASSERT_EQ_CLEANUP("wifi_manager_save_config", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+
+	ret =  wifi_manager_get_config(&gconfig);
+	TC_ASSERT_EQ_CLEANUP("wifi_manager_get_config", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+
+	ret =  wifi_manager_remove_config();
+	TC_ASSERT_EQ_CLEANUP("wifi_manager_remove_config", ret, WIFI_MANAGER_SUCCESS, wifi_manager_deinit());
+
```

Comment:
I think you should call wifi_manager_get_config() again to check that wifi manager removes the configration well.

Reply:
I alraedy created one more Testcase for this scenario. 
Please check  testcase "itc_wifi_manager_remove_get_config_p"

Reply:
Both Testcase are fordifferent purpose.
Like First TC is just for positive scenario likeideally it shopuld work but in second Testcase we arev also checking for scenario in whiich after removing again i called get config which is a negative scenario and use for checking invalid parameter issue also.
please suggest if you are not agree on my above comment.

Reply:
itc_wifi_manager_remove_get_config_p(void) like below
1) save ap config 2) get ap config 3) remove ap config 4) get ap config
first step 3 are exactly same to itc_wifi_manager_get_remove_config_p() with same input parameter.  So I asked you that itc_wifi_manager_remove_get_config_p() could cover it.